pkg/logproto/logproto.proto

```diff
@@ -27,12 +27,11 @@ message PushResponse {
 }
 
 message QueryRequest {
-  string query = 1;
+  string selector = 1;
   uint32 limit = 2;
   google.protobuf.Timestamp start = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
   google.protobuf.Timestamp end = 4 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
   Direction direction = 5;
-  string regex = 6;
```

Comment:
Will removing this break clients if we add a new field `6`? If so, we should mark it as reserved:

```
reserved 6;
```

Reply:
this is querier <-> ingester communication only, unless someone is running a different version of both, it shouldn't be a problem.

Reply:
@cyriltovena that makes sense, although my concern is that we'll run into this when we roll out the new ingesters since it takes a while to complete. I trust your judgment here if you don't think it'll be a problem, though

Reply:
I'll add the reserved, it's not a big deal and I have other things to change.